recipe/meta.yaml

```
@@ -10,15 +10,15 @@ source:
     sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
   skip: true  # [win]
 requirements:
   host:
     - python
     - pip
     - pyjulia >=0.6.0
     - julia
-    - numpy >=1.14.0
+    - numpy
```

If you want to go with option one, you should not revert the changes here, but _additionally_ add the following:
```suggestion
    - numpy
    # temporarily rebuilding with newest numpy because PyCall.jl ends up
    # reinstalling it and we want to avoid spamming ClobberWarnings
    # TODO: remove this as soon as numpy does not get spuriously installed here anymore
    - numpy >=1.24.3
```

---

(I think it was a typo; let me undo this...)

---

No, it was not a typo, that's exactly why I said "additionally".

One line (the unpinned one) is for being up correctly by our infrastructure (conda smithy etc.) as I explained above. The other line is a temporary, additional constraint.

---

I even emphasised _additionally_...

---

My bad. I saw the jobs failing due to requirement errors: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=718646&view=logs&jobId=4f922444-fdfe-5dcf-b824-02f86439ef14&j=4f922444-fdfe-5dcf-b824-02f86439ef14&t=b2a8456a-fb11-5506-ca32-5ccd32538dc0 after committing this initial suggestion and just assumed it was a typo.